tests/ui_tests/app/page/common/tooltip_links/page.py

```diff
@@ -51,6 +51,30 @@ def run_action(self, tab_name: str, action_name: str):
         self.find_and_click(ActionDialog.run)
         self.wait_element_hide(ActionDialog.body)
 
+    @allure.step("Check action {action_name} in ADCM tab is inactive")
+    def is_adcm_actions_inactive(self, action_name: str):
```

Comment:
As you check one aciton, I suggest naming it `is_adcm_action_inactive`

Is ADCM action button unique for ADCM?
I mean we have the similar button on detailed Cluster page, for example

Reply:
no, they are different

Reply:
renamed